### PR TITLE
Add a Picard index Data Manager.

### PR DESCRIPTION
**Pull request by blankenberg**

### CODE REVIEW

File: data_managers/data_manager_picard_index_builder/data_manager/picard_index_builder.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            sequence_name = dbkey
+    return sequence_id, sequence_name
+
+def build_picard_index( data_manager_dict, fasta_filename, params, target_directory, dbkey, sequence_id, sequence_name, data_table_name=DEFAULT_DATA_TABLE_NAME ):
+    #TODO: allow multiple FASTA input files
+    fasta_base_name = os.path.split( fasta_filename )[-1]
+    prefix = os.path.splitext( fasta_base_name )[0]
+    sym_linked_fasta_filename = os.path.join( target_directory, fasta_base_name )
+    os.symlink( fasta_filename, sym_linked_fasta_filename )
+    args = [ 'java', '-Xmx4G', '-jar', os.path.join( os.environ[ 'JAVA_JAR_PATH' ], 'CreateSequenceDictionary.jar' ), 'R=%s' % sym_linked_fasta_filename, 'O=%s.dict' % prefix ]
```
Comment (bgruening) [inline data_managers/data_manager_picard_index_builder/data_manager/picard_index_builder.py]: @blankenberg is it save to hard code these values? Alternatively, we could set `-Xmx` vie `job_conf.xml`.

**Comment by bgruening** [comment]
I'm 100% sure if the Galaxy coding style applies here, but this could be improved. Especially, because I mostly copy and paste @blankenberg data managers to create my own ;)

:+1:

File: data_managers/data_manager_picard_index_builder/data_manager/picard_index_builder.py (hunk 1 above)
Comment (blankenberg) [inline data_managers/data_manager_picard_index_builder/data_manager/picard_index_builder.py]: Having the memory,etc settings configurable would be better for sure.

**Comment by martenson** [comment]
@blankenberg do you want to add anything to this before I merge it?

**Pull request merged**